Traceback (most recent call last):
  File "py/in_python_code.py", line 2, in <module>
    print(a)
NameError: name 'a' is not defined
